wrappers/javascript/shared/src/IndyCredx.ts

```diff
@@ -3,26 +3,26 @@
 
 import type { ObjectHandle } from './ObjectHandle'
 
-export type CredentialEntry = {
+export type NativeCredentialEntry = {
```

Comment:
Any reason for using the `Native` prefix here?

Reply:
This naming was borrowed from Python wrapper when using types that include ObjectHandles (e.g. [RevocationConfig](https://github.com/animo/indy-shared-rs/blob/e152d08233a5dfa17f665554f09215793c668e17/wrappers/python/indy_credx/types.py#L667). IMHO I'm not convinced of this naming, just wanted to differentiate from the ones used in the recently defined API.

Reply:
Ah yeah okay. Again my lack of knowledge on the Python wrapper does not help here, but I would like to keep the wrapper as unopinionated as possible. So whatever indy-credx sends us, next to some casing and deserialisation changes I would not like to change things. If this does not happen I am okay with the naming like this or without the `Native` prefix.